romeo/src/state.rs

```diff
@@ -469,13 +473,44 @@ impl State {
 			State::Initialized {
 				deposits,
 				withdrawals,
+				stacks_block_height,
 				..
 			} => {
```

Comment:
```suggestion
			} => {
    /// The delay in blocks between receiving a deposit request and
    /// creating the deposit transaction.
    const MINT_TRANSACTION_DELAY_BLOCKS: u32 = 1;

```

Reply:
Following the constant standards in the other files,  I think it makes more sense to leave this where it is:

[`BLOCK_POLLING_INTERVAL` in the `stacks_client`](https://github.com/stacks-network/sbtc/blob/ebdcc7bc44aac6628bff8c9b8b5924f0589c0be7/romeo/src/stacks_client.rs#L34)